docs/testing/debug-add-ins-using-devtools-edge-legacy.md

```diff
@@ -57,12 +57,24 @@ To determine which browser you're using, see [Browsers used by Office Add-ins](.
 
 If your add-in uses the Office Dialog API, the dialog runs in a separate process from the task pane (if any) and the tools must attach to that process. Follow these steps.
 
-1. Run the add-in and the tools. 
+1. Run the add-in and the tools.
 1. Open the dialog and then select the **Refresh** button in the tools. The dialog process is shown. Its name comes from the `<title>` element in the HTML file that is open in the dialog.
 1. Select the process to open it and debug just as described in the section [Debug a task pane add-in using Microsoft Edge DevTools Preview](#debug-a-task-pane-add-in-using-microsoft-edge-devtools-preview).
 
    :::image type="content" source="../images/edge-devtools-with-add-in-and-dialog-processes.png" alt-text="Screenshot of Edge DevTools showing a process named My Dialog.":::
 
-## Install a version of Office that uses Microsoft Edge Legacy
+## Switch to the Edge Legacy webview
 
-[!INCLUDE [Steps to install Office that uses Edge Legacy or Internet Explorer](../includes/install-office-that-uses-legacy-edge-or-ie.md)]
\ No newline at end of file
+There are two ways to switch the Edge Legacy webview. You can run a simple command in a command prompt, or you can install a version of Office that uses Edge Legacy by default. We recommend the first method. But you should use the second in the following scenarios:
+
+- Your project was developed with Visual Studio and IIS. It isn't nodejs-based.
+- You want to be absolutely robust in your testing.
+- If for any reason the command line tool doesn't work.
+
+### Switch on the command line
```

Comment:
I'm not sure about saying "on" - it sounds like turning on the command line which I don't think is the intention. Maybe "via" or "using"...

Here and other instances

```suggestion
### Switch via the command line
```